lib/CL/clCreateKernel.c

```diff
@@ -45,6 +46,7 @@ POname(clCreateKernel)(cl_program program,
   int errcode;
   int error;
   unsigned device_i;
+  poclcc_global poclcc;
```

Comment:
This is a temporary variable needed for the functions that will get the binary of the kernel to be created.
I didn't put it in the _"if"_ statement where it is used because it needs to be free is case of error. 
If you want if to be in the _"if"_ statement, I will remove the _POCL_GOTO_ERROR_COND_ and manage error (part of it) in the _"if"_ statement.